scraper/dev/docker/Dockerfile.base

```diff
@@ -10,10 +12,16 @@ RUN useradd -d /home/seleuser -m seleuser
 RUN chown -R seleuser /home/seleuser
 RUN chgrp -R seleuser /home/seleuser
 
-RUN apt-get -y update
-RUN apt-get install -y -q software-properties-common wget
+RUN apt-get update -y && apt-get install -yq \
+    software-properties-common
 RUN add-apt-repository -y ppa:openjdk-r/ppa
-RUN wget -qO- https://deb.nodesource.com/setup_4.x | sudo bash -
+RUN apt-get update -y && apt-get install -yq \
```

Comment:
Lines 15 and 18 both do a `apt-get update` and `apt-get install`. Maybe merge them into one?

Reply:
No because one layer is created for every command. These apt-get are not done at the same time and on the same environment. A deeper layer is not updating an upper one.

Reply:
Ok